code/modules/assembly/flash.dm

```diff
@@ -126,7 +126,9 @@
 				to_chat(M, "<span class='userdanger'>[user] blinds you with the flash!</span>")
 			else
 				to_chat(M, "<span class='userdanger'>You are blinded by [src]!</span>")
-			M.Paralyze(rand(80,120))
+			M.adjustStaminaLoss(rand(20,50))
+			if(prob(40))

```

Comment:
Great. Even more rng

Reply:
prob(80)
still less RNG on the stuntime on the original flashcode